tests/dummy/app/templates/demos/sl-radio.hbs

```diff
@@ -34,27 +34,27 @@
 <h3>Properties</h3>
 
 <div class="list-group">
-    {{#property-text name="disabled" type="boolean" default="false"}}
+    {{#property-text name="disabled" type="Boolean" default="false"}}
         When true, the radio button will be disabled.
     {{/property-text}}
 
-    {{#property-text name="inline" type="boolean" default="false"}}
+    {{#property-text name="inline" type="Boolean" default="false"}}
         Display the radio inputs inline with each other.
     {{/property-text}}
 
-    {{#property-text name="label" type="string"}}
+    {{#property-text name="label" type="String"}}
         Text label for the button.
     {{/property-text}}
 
-    {{#property-text name="name" type="string"}}
+    {{#property-text name="name" type="String"}}
         <code>name</code> attribute value for the button.
     {{/property-text}}
 
-    {{#property-text name="readonly" type="boolean" default="false"}}
+    {{#property-text name="readonly" type="Boolean" default="false"}}
         When true, the radio button will be readonly.
     {{/property-text}}
 
-    {{#property-text name="value" type="mixed"}}
+    {{#property-text name="value" type="*"}}
```

Comment:
Wouldn't `sl-radio` and `sl-radio-group` be the same types?  Could see it being for sure a `String` because of the HTML value attribute, could also see it being a `Number` because that's what could be passed into it from the JS side.  Either way, seems they should be consistent with one another.